### PR TITLE
🌅  Write intersphinx references

### PR DESCRIPTION
**Pull request by rowanc1**
Intersphinx CLI:

image

### CODE REVIEW

File: packages/myst-cli-utils/package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  "description": "Utils for logging, error messages and session management",
+  "author": "Rowan Cockett <rowan@curvenote.com>",
+  "homepage": "https://github.com/curvenote/curvenote#readme",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "exports": {
```
Comment (fwkoch) [inline packages/myst-cli-utils/package.json:14]: We had to take this block out of `myst-common` to get builds working

File: apps/cli/src/store/local/actions.ts [hunk 2]
```diff
@@ -448,6 +451,48 @@ export async function writeSiteManifest(session: ISession) {
...
+  session: ISession,
+  inv: Inventory,
+  opts: { projectPath: string },
+) {
+  const { pages } = loadProject(session, opts.projectPath);
+  const pageReferenceStates = selectPageReferenceStates(session, pages);
+  pageReferenceStates.forEach((page) => {
+    const { title } = selectors.selectFileInfo(session.store.getState(), page.file);
+    inv.setEntry({
+      type: 'std:doc',
```
Comment (fwkoch) [inline apps/cli/src/store/local/actions.ts]: Looks like these types are from `intersphinx` - `enum` maybe?

**Review by fwkoch — APPROVED** [review]
Not a super detailed review, but everything in here makes sense, the content of `objects.inv` seems straightforward.

File: apps/cli/src/store/local/actions.ts (hunk 2 above)
Comment (rowanc1) [reply apps/cli/src/store/local/actions.ts]: Done.

File: packages/myst-cli-utils/package.json (hunk 1 above)
Comment (rowanc1) [reply packages/myst-cli-utils/package.json:14]: Will leave it in and note it for later?

**Pull request merged**